app/views/state_file/state_file_pages/_footer.html.erb

```diff
@@ -15,14 +15,20 @@
       <ul class="grid__item width-one-fourth footer-links">
         <% unless before_state_file_launch? %>
           <li>
-            <%= link_to t(".privacy_policy"), state_file_privacy_policy_path %>
+            <%= mail_to "help@fileyourstatetaxes.org", t("general.contact_us")  %>
           </li>
 
-          <li><%= link_to t(".about"), root_path %></li>
+          <li>
+            <%= link_to t(".privacy_policy"), state_file_privacy_policy_path %>
+          </li>
 
           <li><%= link_to t(".faq"), state_faq_path(us_state: 'us') %></li>
 
-          <li><%= link_to t(".terms_of_service"), "" %></li>
+          <li>
+            <% if I18n.locale != :es %>
+              <%= link_to_spanish %>
+            <% end %>
+          </li>
```

Comment:
should we add English option if in Spanish locale like we do in the header?
```suggestion
           <% if I18n.locale != :en %>
              <%= link_to_english %>
            <% end %>
            </li>
```

Reply:
such a good catch! fixed it